storage/engine/rocksdb/db.cc

```diff
@@ -1077,9 +1077,15 @@ class BaseDeltaIterator : public rocksdb::Iterator {
 }  // namespace
 
 DBStatus DBOpen(DBEngine **db, DBSlice dir, DBOptions db_opts) {
+  // Divide the cache space into two levels: the fast row cache
+  // and the slower but more space-efficient block cache.
+  // TODO(bdarnell): do we need separate settings for these?
+  const auto row_cache_size = db_opts.cache_size / 2;
+  const auto block_cache_size = db_opts.cache_size / 2;
```

Comment:
No, I haven't done any experimentation. If the row cache is used for all operations and not just point lookups (I haven't confirmed this but the scan speedups point in this direction), it might make sense to go all the way to 100% row cache (it's faster for small working sets, and it's unclear how important medium-sized working sets that could fit in the block cache but not in the row cache are).